content/en/payment.md

```diff
@@ -304,4 +442,390 @@ await swell.cart.update({
 })
 ```
 
+## Manual payment integrations
+
+If you need more flexibility in the integration process, you can use the following guides to achieve this.
+
+### Stripe + Klarna, iDEAL & Bancontact integration
+
+This guide describes how to integrate Klarna, iDEAL and Bancontact using Stripe sources with a custom checkout flow.
+
+Start by connecting a Stripe account to Swell (Settings > Payments); open "Advanced options" under Stripe settings and check `Enable Klarna` / `Enable iDEAL` / `Enable Bancontact`.
+
+#### Using Stripe.js
+
+​
+[Include Stripe.js](https://stripe.com/docs/js/including) on the payment page of your site. This can be done in two ways:
+
+- Load directly from `https://js.stripe.com`.
+
+```html
+<script src="https://js.stripe.com/v3/"></script>
+```
+
+- Stripe provides an [npm package](https://github.com/stripe/stripe-js) to load and use Stripe.js as a module.
+
+#### Initializing Stripe.js
+
+​
+Once loaded, [initialize](https://stripe.com/docs/js/initializing) it with publishable key.
+
+If loaded from a script:
+
+```js
+// Client side
+const stripe = Stripe('pk_test_...')
+```
+
+​
+If loaded from a JS module:
+
+```js
+// Client side
+import { loadStripe } from '@stripe/stripe-js'
+const stripe = await loadStripe('pk_test_...')
+```
+
+#### Install Stripe server-side library
+
+​
+This library will be used to access the [Stripe API](https://stripe.com/docs/api) and create payment intents. To install the [npm package](https://www.npmjs.com/package/stripe):
+
+```
+npm i --save stripe
+```
+
+​
+Initialize Stripe with a secret key:
+
+```js
+// Server side
+const stripe = require('stripe')('sk_test_...')
+```
+
+#### iDEAL integration
+
+#### Create iDEAL element
+
+Using Stripe elements:
+
+```js
+// Client side
+const stripe = Stripe('pk_test_...');
+​
+const elements = stripe.elements();
+const ideal = elements.create('idealBank', options?);
```

Comment:
@vstudenichnik-insoft is this '?' needed here?
`options?`

Reply:
`?` means this is an optional parameter

Reply:
@vstudenichnik-insoft , are you just indicating that the parameter is optional in a sudo-code fashion? Wouldn't this throw an error if the user included it in the code?

Reply:
Perhaps we could use a comment instead?
`const ideal = elements.create('idealBank', options /* optional param */);`

Reply:
@greph  I think any developer would understand this, but I updated it anyway this way:
`const ideal = elements.create('idealBank', options); // 'options' is an optional object`

Reply:
Thanks @vstudenichnik-insoft ! It's common to copy/paste code so just want to make sure the code is valid.